src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilsTest.java

```diff
@@ -359,6 +359,6 @@ public void testGetPreviousSibling() {
     @Test
     public void testGetTokenNames() {
         assertEquals("Unexpected token name",
-            "HTML_COMMENT", JavadocUtils.getTokenName(20078));
+            "HTML_COMMENT", JavadocUtils.getTokenName(20081));
```

Comment:
we can not change ID of tokens.
It will seriously damage compatibility.
Please explain this change.

Reply:
I create issue #5139 .
@ps-sp , please fix it first, and rebase this PR after fix, to make sure we are not going to damage a lot of stuff.